extensions/amp-date-picker/0.1/dates-list.js

```diff
@@ -97,6 +136,42 @@ export class DatesList {
       return DateType.DATE;
     }
 
+    const dateStr = /** @type {string} */ (date);
+    if (tryParseRrulestr(dateStr)) {
+      return DateType.RRULE;
+    }
+
     return DateType.INVALID;
   }
 }
+
+/**
+ * RRULE returns dates as local time formatted at UTC, so the
+ * Date.prototype.getUTC* methods must be used to create a new date object.
+ * {@link https://github.com/jakubroztocil/rrule#important-use-utc-dates}
+ * @param {!Date} rruleDate
+ * @return {!Date}
+ */
+function normalizeRruleReturn(rruleDate) {
+  const year = rruleDate.getUTCFullYear();
+  const month = rruleDate.getUTCMonth();
+  const day = rruleDate.getUTCDate();
+  const hours = rruleDate.getUTCHours();
+  const minutes = rruleDate.getUTCMinutes();
+  const seconds = rruleDate.getUTCSeconds();
+  const ms = rruleDate.getUTCMilliseconds();
+  return new Date(year, month, day, hours, minutes, seconds, ms);
+}
+
+/**
+ * Tries to parse a string into an RRULE object.
+ * @param {string} str A string which represents a repeating date RRULE spec.
+ * @return {?RRule}
+ */
+function tryParseRrulestr(str) {
+  try {
+    return rrulestr(str);
```

Comment:
The latest version of `RRule` has two args for `rrulestr`, resulting in this error:
```js
extensions/amp-date-picker/0.1/dates-list.js:173: ERROR - [JSC_WRONG_ARGUMENT_COUNT] Function rrulestr$$module$node_modules$rrule$dist$esm$src$rrulestr: called with 1 argument(s). Function requires at least 2 argument(s) and no more than 2 argument(s).
    return rrulestr(str);
           ^^^^^^^^^^^^^
```

I think you can call `rrulestr(str, {})` to get past the error.